### ISSUE TITLE
Can not click into MediumEditorField in FF to give it the focus

### ISSUE THREAD

**Issue by moberwasserlechner** [moberwasserlechner/vaadin-medium-editor#13]
Field only gets the focus, if I tab into it.

**Comment by mstahv** [comment]
Hi, just to note that my FF related issue was apparently due an older version of FF. Updated and it works fine in FF. I hope you can solve the Safari issue still!

**Comment by moberwasserlechner** [comment]
@mstahv thanks for clarification. I fixed the Safari problem in #12.

**Comment by moberwasserlechner** [comment]
This on the other hand was due to a style (min-height) I set to the MediumEditorField. The inner component did not adapt to that height although I setSizeFull(), so I simply had to set the style on the editor too.

```
    MediumEditorField content = new MediumEditorField(this.i18n.get("core.content"));
    content.addStyleName("editorStyle");

    MediumEditor editor = content.getEditor();
    editor.addStyleName("editorStyle");
```

I don't think I can handle this kind of problem in the component, because its very special to the current situation the addon is used.
